internal/context/init.go

```diff
@@ -34,16 +35,21 @@ func Init(projectName string, outDir string) *projectconfig.ZeroProjectConfig {
 	}
 
 	projectConfig := defaultProjConfig(projectName)
-	projectConfig.Name = promptProjectName(projectName)
-	projectConfig.Context["ShouldPushRepoUpstream"] = promptPushRepoUpstream()
-	projectConfig.Context["GithubRootOrg"] = promptGithubRootOrg()
-	projectConfig.Context["githubPersonalToken"] = promptGithubPersonalToken(projectName)
-
-	// chooseCloudProvider(&projectConfig)
-	// fmt.Println(&projectConfig)
-	// s := project.GetSecrets(rootDir)
-	// fillProviderDetails(&projectConfig, s)
-	// fmt.Println(&projectConfig)
+
+	prompts := getProjectPrompts(projectName)
+
+	projectConfig.Name = prompts["projectName"].GetParam(projectConfig.Parameters)
+
+	projectConfig.Parameters["ShouldPushRepoUpstream"] = prompts["ShouldPushRepoUpstream"].GetParam(projectConfig.Parameters)
```

Comment:
You don't need to do this as part of this PR but shouldn't these be part of the projectConfig struct rather than parameters? Parameters are passed into project templates, but these values we will be using for zero itself during the create phase.

Reply:
I'll do this as part of my ticket.